src/swmmanywhere/utilities.py

```diff
@@ -49,3 +55,150 @@ def yaml_dump(o: Any, stream: Any = None, **kwargs: Any) -> str:
         sort_keys=False,
         **kwargs,
     )
+
+
+def plot_basic(model_dir: Path):
```

Comment:
I would put all of these plotting tools into its own `plot.py` module, and possibly break down some of them into smaller chunks, but maybe leave it for a future time - or Hacktoberfest!!

Reply:
now in #326